siema.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title> tytuł strony </title>
+        <meta name="description" content="This is the description">
+    </head>
+    <body>
+        <header>
+            <nav>
+                <ul> 
+                    <li><a href="siema.html">HOME</a></li>
+                    <li>STORE</li>
+                    <li>ABOUT</li>
+                </ul>
+            </nav>
+            <hr>
+            <h1>NIE ŚMIEJMY SIĘ JUŻ LOL</h1>
+            <button type="button">
+                MIŁOŚĆ
+            </button>
+            <br>
```

Comment:
Try not using `<br />` tags. Instead, you could add some CSS once you know the essentials. Here's an example
```
<! -- Place <style> inside the <head> or in a css file and link it -->
<style>
.button--love {
  margin-bottom: 1rem;
}
</style>

<button class="button--love" type="button">
  MIŁOŚĆ
</button>
```
```